ext/openssl/openssl.stub.php

```diff
@@ -19,7 +19,7 @@ function openssl_x509_export_to_file(OpenSSLCertificate|string $certificate, str
 /** @param string $output */
 function openssl_x509_export(OpenSSLCertificate|string $certificate, &$output, bool $no_text = true): bool {}
 
-function openssl_x509_fingerprint(OpenSSLCertificate|string $certificate, string $digest_algorithm = "sha1", bool $raw_output = false): string|false {}
+function openssl_x509_fingerprint(OpenSSLCertificate|string $certificate, string $digest_algo = "sha1", bool $raw_output = false): string|false {}
```

Comment:
```suggestion
function openssl_x509_fingerprint(OpenSSLCertificate|string $certificate, string $digest_algo = "sha1", bool $binary = false): string|false {}
```
Etc. I would suggest updating this everywhere (it's also in basic_functions.stub.php) to make sure we don't forget.

Reply:
Oh no, I unintentionally committed this file. But I love the suggested change, I didn't notice openssl also has `$raw_output`.